ucsb-cs156-m23/proj-happycows-m23-10am-4
# FIX: Removed 404 Error Page on Navigation and Reload

## Overview

In this PR, we remove the issue of the 404 Error screen appearing briefly whenever a page is loaded or reloaded. We accomplish this by rendering a blank page until the correct one has been served to the client. This also has a secondary benefit of reducing page loading times, as it removes the initial React Router query (Which resulted in 404).

## Screenshots (Optional)

*Insert absence of 404 Error*

## Future Possibilities (Optional)

Some possible points could be:
- Rather than serving a blank page, we could maintain the current page (if applicable) until the new one is ready.
- It could also be nice to refactor the app to use the newer React data routers, which have some nice features for mitigating similar issues

## Validation (Optional)

1. Deploy this branch on dokku (localhost is too fast to replicate the issue).
2. Navigate to a page other than root 
3. Reload the page repeatedly and make sure “404” never appears

## Tests


## Linked Issues

Closes #33
